lib/build/build.go

```diff
@@ -79,6 +88,11 @@ func LongVersionFor(program string) string {
 	// This string and date format is essentially part of our external API. Never change it.
 	date := Date.UTC().Format("2006-01-02 15:04:05 MST")
 	v := fmt.Sprintf(`%s %s "%s" (%s %s-%s) %s@%s %s`, program, Version, Codename, runtime.Version(), runtime.GOOS, runtime.GOARCH, User, Host, date)
+	for _, envVar := range envTags {
+		if os.Getenv(envVar) != "" {
+			Tags = append(Tags, strings.ToLower(envVar))
```

Comment:
Do we really want this as tags? Tags are usually good for filtering, this is probably just "extra" data?